src/data/metadata.csv

```diff
@@ -2294,7 +2294,14 @@ LS,LS,Lesotho,,,,,,-1
 LT,LT,Lithuania,,,,,,-1
 LU,LU,Luxembourg,,,,,,-1
 LV,LV,Latvia,,,,,,-1
-LY,LY,Libya,,,,,,-1
+LY_BA,LY,Libya,BA,Benghazi,,,Benghazi,,,,
+LY_JI,LY,Libya,JI,Jafara,,,Al-Jafara,,,,
+LY_MS,LY,Libya,MS,Misrata District,,,Misrata,,,,
+LY_SB,LY,Libya,SB,Sabha District,,,Sabha,,,,
+LY_ST,LY,Libya,ST,Sabratha District,,,Sabratha,,,,
+LY_TB,LY,Libya,TB,Tripoli District,,,Tripoli,,,,
+LY_WS,LY,Libya,WS,Wadi al Shatii District,,,Wadi al Shati,,,,
+LY_ZL,LY,Libya,ZL,Zliten District,,,Zliten,,,,
```

Comment:
It looks like there is the wrong number of fields here. For instance I think the first line should be:
`LY_BA,LY,Libya,BA,Benghazi,,,,Benghazi,`

Note that if the subregion name is the same as the string_match field, you can skip adding the string_match (although having it does not hurt).

Reply:
Done,  I've updated the number of columns to be back to what they were :)